dev/NavigationView/NavigationView.cpp

```diff
@@ -1358,6 +1362,36 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    auto newButtonWidths = [this]()
+    {
+        if (DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+        {
+            return static_cast<double>(c_paneToggleButtonWidth);
+        }
+        return CompactPaneLength();
+    }();
+
+    if (const auto backButton = m_backButton.get())
```

Comment:
>auto [](start = 14, length = 4)

auto&& for locals to tracker_ref

Reply:
Still missed this one.

---
In reply to: [401175089](https://github.com/microsoft/microsoft-ui-xaml/pull/2051#discussion_r401175089) [](ancestors = 401175089)